app/services/video_service.rb

```diff
@@ -1,32 +1,22 @@
 class VideoService
-  def get_videos(category)
-    # get_json('/api/v1/videosfind_all', category)
-    response = if category == 'motivational'
-                 File.read('lib/mock/motivational_video.json')
-               elsif category == 'meditation'
-                 File.read('lib/mock/meditation_video.json')
-               else
-                 File.read('lib/mock/yoga_video.json')
-               end
-    JSON.parse(response, symbolize_names: true)
+  def all_videos
+    get_json('/api/v1/videos')
   end
 
   def get_suggested_videos(mood)
-    # get_json("/api/v1/suggestions?mood=#{mood}")
-    response = File.read(Rails.root.join('lib', 'mock', "#{mood}_videos.json"))
-    JSON.parse(response, symbolize_names: true)
+    get_json("/api/v1/videos/#{mood}")
   end
 
-  # private
-  #
-  # def get_json(url, params = nil)
-  #   response = conn.get(url, params)
-  #   JSON.parse(response.body, symbolize_names: true)
-  # end
-  #
-  # def conn
-  #   Faraday.new('https://localhost:3000') do |conn|
-  #     conn.adapter Faraday.default_adapter
-  #   end
-  # end
+  private
+
+  def get_json(url)
+    response = conn.get(url)
+    JSON.parse(response.body, symbolize_names: true)
+  end
+
+  def conn
+    Faraday.new(ENV['MICROSERVICE_DOMAIN']) do |conn|
```

Comment:
This looks good - we will need to set this environmental variable to the microservice heroku domain in Travis CI and Heroku

Reply:
Already done.